src/constants.ts

```diff
@@ -1,8 +1,10 @@
 import { SupportedEventsU } from "./types";
 
-export const notifyTriggers: Record<string, SupportedEventsU> = {
-    "payment": "issue_comment.created",
-    "reminder": "issue_comment.created",
-    "disqualification": "issues.unassigned",
-    "review": "pull_request.review_requested",
-}
\ No newline at end of file
+export type NotificationTriggers = "payment" | "review" | "disqualification" | "reminder";
```

Comment:
I suppose this pull won't handle my request on it being dictated by the user.

Reply:
No the handlers for comment.created & edited will need completely re-done and a proper interface designed for it. It has arms and legs but doable and sounds good for plugin devs